AUTHORS

```diff
@@ -3523,5 +3523,10 @@ Shrujal Shah <shrujalshah@hotmail.com>
 Taha-Chaudhry <46199675+Taha-Chaudhry@users.noreply.github.com>
 smitley <brandon.smitley@gmail.com>
 Brian Muenzenmeyer <Brian.Muenzenmeyer@target.com>
+sidwebworks <xodedeveloper@gmail.com>
+Connor Burton <me@connorburton.com>
+chexiongsheng <chexiongsheng@qq.com>
+Lucas Santos <lhs.santoss@gmail.com>
+“Pooja <pakcutie14@gmail.com>
```

Comment:
@Pooja444 Is the punctuation mark at the start of your name an error? If it was unintentional, we can get rid of it with a `.mailmap` entry. But if your name is appearing as intended, then of course, I won't do that.